### DISCUSSION TITLE
Issues in regards to enabling 'Calculator plugin' correctly

### DISCUSSION THREAD

**Discussion by GitTimeraider** [searxng/searxng#3529] [Q&A] [answered]
I am trying to have basic calculator enabled by default but it does not seem to work.
My settings.yml part of enabled plugins looks like this:
```
enabled_plugins:
#   # these plugins are enabled if nothing is configured ..
#   - 'Hash plugin'
#   - 'Self Information'
  - 'Tracker URL remover'
#   - 'Ahmia blacklist'  # activation depends on outgoing.using_tor_proxy
#   # these plugins are disabled if nothing is configured ..
#   - 'Hostname replace'  # see hostname_replace configuration below
  - 'Calculator plugin'
#   - 'Open Access DOI rewrite'
#   - 'Tor check plugin'
```

What am I doing wrong?

**Comment by GitTimeraider** [comment, marked as answer]
Ok.. turns out the Calculator plugin in fact is named Basic Calculator and not Calculator Plugin.
After changing the text Calculator Plugin into Basic Calculator it worked.

**Reply by return42** [reply]
Thanks for pointing out / fixed in https://github.com/searxng/searxng/pull/3530

Further note, the plugin is not "allowed" on public instances (follow discussion at https://github.com/searxng/searxng/pull/3377)

https://github.com/searxng/searxng/blob/0fa81fc782feb104bbd6616d87a6f441aad7d5bd/searx/plugins/calculator.py#L87-L88

https://github.com/searxng/searxng/blob/0fa81fc782feb104bbd6616d87a6f441aad7d5bd/searx/plugins/calculator.py#L54-L57

FWIW: I patched it in my https://github.com/return42/searxng/commits/darmarit.org/ branch (82b301c1)
